azurePipelines.yml

```diff
@@ -0,0 +1,43 @@
+variables:
+- name: zuluLinuxLatest
+  value: '/usr/lib/jvm/zulu-11-azure-amd64' 
+trigger:
+  batch: true
+  branches:
+    include:
+      - '*'
+pr:
+  branches:
+    include:
+      - '*'
+jobs:
+  - job: Build
+    pool:
+      vmImage: 'ubuntu-16.04'    
+    steps:
+      - checkout: self
+        clean: true
+      - bash: |
+          git config --global user.email "mastercoms@tutanota.de"
+          git config --global user.name "mastercoms"  
+        displayName: Configure Git
+      - bash: ./glowkit p
+        displayName: Apply Patches
+      - bash: (cd Glowkit-Patched && mvn -T 1C -B dependency:go-offline)
```

Comment:
This step was being used for caching and is not needed in Azure at the moment.

Reply:
Commented out, but I think we should keep the code handy. Caching is likely coming in May or June.